### PR TITLE
Generate ansible burstbuffer RPM 

### PR DESCRIPTION
**Pull request by meahoibm**
Generate ibm-ansibleburstbuffer RPM (ibm-burstbuffer-ansible would be erased by a playbook doing yum erase ibm-burstbuffer*).

### CODE REVIEW

File: bb/CMakeLists.txt [hunk 1]
```diff
@@ -26,6 +30,7 @@ Prefix: ${CMAKE_INSTALL_PREFIX}/bb/scripts
 set(CPACK_RPM_burstbuffer_PACKAGE_REQUIRES       "nvme-cli,nvmetcli,fuse,fuse-libs,util-linux,lvm2,coreutils,xfsprogs,openssh-clients,grep,perl-Sys-Syslog,sysstat" PARENT_SCOPE)
 set(CPACK_RPM_burstbuffer-tests_PACKAGE_REQUIRES "ibm_smpi > 10.1"   PARENT_SCOPE)
 set(CPACK_RPM_burstbuffer-tools_PACKAGE_REQUIRES "ibm_smpi > 10.1"   PARENT_SCOPE)
+set(CPACK_RPM_ansibleburstbuffer_PACKAGE_REQUIRES "ansible >= 2.9.6"   PARENT_SCOPE)
```
Comment (tgooding) [inline bb/CMakeLists.txt:33]: This line looks incorrect.  Think it should be CPACK_RPM_burstbuffer-ansible_PACKAGE_REQUIRES

Comment (meahoibm) [reply bb/CMakeLists.txt:33]: Generate ibm-ansibleburstbuffer RPM (ibm-burstbuffer-ansible would be erased by a playbook doing yum erase ibm-burstbuffer*).

**Comment by tgooding** [comment]
Should the ansible RPM be marked noarch?  e.g.,:
> SET(CPACK_RPM_burstbuffer-ansible_PACKAGE_ARCHITECTURE "noarch")

**Comment by meahoibm** [comment]
> Should the ansible RPM be marked noarch? e.g.,:
> 
> > SET(CPACK_RPM_burstbuffer-ansible_PACKAGE_ARCHITECTURE "noarch")

Adding noarch

**Pull request merged**